frappe/model/document.py

```diff
@@ -152,6 +152,17 @@ def load_from_db(self):
 			super(Document, self).__init__(d)
 
 		for df in self._get_table_fields():
+			# Make sure not to query the DB for a child table, if it is a virtual one.
+			# During frappe is installed, the property "is_virtual" is not available in tabDocType, so
+			# we need to filter those cases for the access to frappe.db.get_value() as it would crash otherwise.
+			if (
+				hasattr(self, "doctype")
+				and not hasattr(self, "module")
+				and frappe.db.get_value("DocType", df.options, "is_virtual", cache=True)
```

Comment:
Would this work instead? Based on the other ways we've checked whether a DocType is virtual in this PR

```suggestion
				and frappe.get_meta(df.options).is_virtual == 1
```

Reply:
@gavindsouza Sure, as I'm beginner, I just copied the matching requests from somewhere in the codebase. If I had knew, there is a leaner API, I'd of course prefer that in the way you suggested.
.. as long as the tests are still passing of course ;-)